src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -11,21 +11,20 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        if (nullCheck(user) && !isUserExist(user)
-                && isAgeCorrect(user) && isPasswordCorrect(user)) {
-            return storageDao.add(user);
-        }
-        return null;
+        nullCheck(user);
+        userExistCheck(user);
+        ageCheck(user);
+        passwordCheck(user);
+        return storageDao.add(user);
```

Comment:
i'm not sure with this
Is this a better solution?

Reply:
I like it more